src/ladds/Simulation.cpp

```diff
@@ -430,6 +442,13 @@ size_t Simulation::simulationLoop(AutoPas_t &autopas,
   return totalConjunctions;
 }
 
+void Simulation::removeParticleSpawnProtection(autopas::AutoPas<Particle> &autopas) {
+  SPDLOG_LOGGER_INFO(logger.get(), "Removing spawn protection for all particles.");
+  for (auto &particle : autopas) {
+    particle.setParentIdentifier(std::numeric_limits<size_t>::max());
+  };
```

Comment:
```suggestion
  autopas.forEach([](auto &particle) { particle.setParentIdentifier(std::numeric_limits<size_t>::max()); });
```
New `for_each` functions are faster and more future proof ;)